src/Halogen/Svg/Attributes/Utils.purs

```diff
@@ -0,0 +1,10 @@
+module Halogen.Svg.Attributes.Utils
+  ( printArray
+  ) where
+
+import Prelude
+import Data.String (joinWith)
+
+-- A helper function. Only used in other modules in Halogen.Svg.Attributes
+printArray :: forall a. Show a => Array a -> String
+printArray xs = joinWith "," (show <$> xs)
```

Comment:
Can this be written using a `foldl`? Otherwise, we're creating a new array and then folding that new array, which is unnecessary work.

Reply:
Actually, I'm wondering if that would be faster since a `foldl` would need to create a new record for each element... 

Please disregard my earlier comment.